README.md

```diff
@@ -44,3 +44,27 @@ module.exports = {
 ```bash
 yarn jest
 ```
+
+
+## Options
+
+This project uses [cosmiconfig](https://github.com/davidtheclark/cosmiconfig), so you can provide config via:
+* a `jest-runner-eslint` property in your `package.json`
+* a `jest-runner-eslint.config.js` JS file
+* a `.jest-runner-eslintrc` JSON file
+
+```js
+module.exports = {
+  "cliOptions": {
+    // ESLint cli options options here
```

Comment:
can you include some examples, especially ones that aren't all lowercase, or have multiple words?